Explain grafana dashboard provider (#835)
codenvy/che-docs: src/main/pages/che-7/administration-guide/proc_viewing-che-metrics-on-grafana-dashboards.adoc

Grafana is run on port `3000` with a corresponding *service* and *route*.

Three ConfigMaps are used to configure Grafana:

* `grafana-datasources` -- configuration for Grafana datasource, a Prometheus endpoint
* `grafana-dashboards` -- configuration of Grafana dashboards and panels

> * `grafana-dashboard-provider`  -- configuration of Grafana dashboard provider API object, that tells Grafana where to look in the filesystem for pre-provisioned dashboards

Reviewer: ```suggestion
* `grafana-dashboard-provider`  -- configuration of the Grafana dashboard provider API object, which tells Grafana where to look in the file system for pre-provisioned dashboards
```
